CatchMe/CatchMe/Source/Cells/Main/characterCVC.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  characterCVC.swift
+//  CatchMe
+//
+//  Created by 양수빈 on 2021/07/04.
+//
+
+import UIKit
+
+class characterCVC: UICollectionViewCell {
+
+    static let identifier = "characterCVC"
+    
+    // MARK: - IBOutlet
+    @IBOutlet weak var characterImage: UIImageView!
+    
+    // MARK: - Life Cycle
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        characterImage.backgroundColor = .purple
+        characterImage.translatesAutoresizingMaskIntoConstraints = false
+        characterImage.snp.makeConstraints { make in
+            make.center.equalToSuperview()
+            make.width.height.equalTo(174)
+        }
```

Comment:
```suggestion
     func configUI() {
        characterImage.backgroundColor = .purple
        characterImage.translatesAutoresizingMaskIntoConstraints = false
        characterImage.snp.makeConstraints { make in
            make.center.equalToSuperview()
            make.width.height.equalTo(174)
        }
      }
```
awakeFromNib은 Life Cycle 부분 함수이기 때문에 함수만 넣어줘야 해요! 그래서 저 부분들을 따로 함수로 빼고 그 함수들을 넣어줘야 합니다!

근데 characterImage가 IBOutlet이면 이미 Cell에 레이아웃이 잡혀있지 않나요? 
만약에 코드로 다시 잡은거면 Xib에서 다시 잡아주세용! 스보에서 잡을 수 있는건 최대한 하는 게 좋아요!

Reply:
아마 그렇게 한다면 backgroundColor만 코드로 잡으면 됩니다!